workspaces/tech-insights/plugins/tech-insights-client/src/TechInsightsClient.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  BulkCheckResponse,
+  CheckResult,
+  FactSchema,
+} from '@backstage-community/plugin-tech-insights-common';
+import { Check, InsightFacts } from './types';
+import { DiscoveryApi, IdentityApi } from '@backstage/core-plugin-api';
+import { ResponseError } from '@backstage/errors';
+import {
+  CompoundEntityRef,
+  stringifyEntityRef,
+} from '@backstage/catalog-model';
+import qs from 'qs';
+import { AuthService } from '@backstage/backend-plugin-api';
+
+/** @public */
+export class TechInsightsClient {
```

Comment:
This class is copied except removing the references to render components and the `getToken` function.